website/content/en/docs/olm-integration/user-guide.md

```diff
@@ -65,6 +65,8 @@ catalogsources.operators.coreos.com                          CustomResourceDefin
 ...
 ```
 
+ The default version which is installed in the cluster, can be specified in the `Makefile`. The OLM manifests for the default version specified in the Makefile are pre-fetched and present inside the SDK repository as Go source code. Modifying the `OLM_VERSION` variable in Makefile and running `make olm-manifests` will update the current source code file `internal/olm/olm-bindata-x.x.x`, where `x.x.x` refers to the specified version of OLM manifests.
+
```

Comment:
> The default version which is installed in the cluster, can be specified in the `Makefile`
How users can specified in their project Makefile the default OLM version? It is specified in the SDK's project Makefile and shipped with its binary. Users have not the SDK Makefile. 

Note that end-users has not the `make olm-manifests`. It can be part of the contributions guide but not of the user-guide.

Reply:
Yeah, this is not the right place. Would it be suitable to include this in developers guide instead?

Reply:
The developer guide makes sense to me.